src/probnum/problems/zoo/filtsmooth/_ssm_examples.py

```diff
@@ -0,0 +1,328 @@
+        \end{pmatrix} X(t_{n})
+        + r_n
+
+    where :math:`q(t_n) \sim \mathcal{N}(0, Q)` and :math:`r_n \sim \mathcal{N}(0, R)`
+    for process noise covariance matrix :math:`Q` and measurement noise covariance
+    matrix :math:`R`.
+
+    Parameters
+    ----------
+    delta_t : float
+        The step size of the discretized time grid on which the model is considered.
+    measvar : float
+        The marginal measurement variance.
+    t_max : float
+        The time limit of the grid.
+
+    Returns
+    -------
+    dynmod
+        The `DiscreteLTIGaussian` dynamics model.
+    measmod
+        The `DiscreteLTIGaussian` measurement model.
+    initrv
+        A `Normal` random variable for the Gaussian initial conditions.
+    regression_problem
+        A `RegressionProblem` with time points and noisy observations.
+
+    References
+    ----------
+    .. [1] Särkkä, Simo. Bayesian Filtering and Smoothing. Cambridge University Press,
+        2013.
+
+    """
+
+    dynamat = np.eye(4) + delta_t * np.diag(np.ones(2), 2)
+    dynadiff = (
+        np.diag(np.array([delta_t ** 3 / 3, delta_t ** 3 / 3, delta_t, delta_t]))
+        + np.diag(np.array([delta_t ** 2 / 2, delta_t ** 2 / 2]), 2)
+        + np.diag(np.array([delta_t ** 2 / 2, delta_t ** 2 / 2]), -2)
+    )
+    measmat = np.eye(2, 4)
+    measdiff = measvar * np.eye(2)
+    mean = np.zeros(4)
+    cov = 0.5 * measvar * np.eye(4)
+
+    dynmod = statespace.DiscreteLTIGaussian(
+        state_trans_mat=dynamat, shift_vec=np.zeros(4), proc_noise_cov_mat=dynadiff
+    )
+    measmod = statespace.DiscreteLTIGaussian(
+        state_trans_mat=measmat,
+        shift_vec=np.zeros(2),
+        proc_noise_cov_mat=measdiff,
+    )
+    initrv = randvars.Normal(mean, cov)
+
+    # Generate data
+    times = np.arange(0.0, t_max, step=delta_t)
+    states, obs = statespace.generate_samples(
+        dynmod=dynmod, measmod=measmod, initrv=initrv, times=times
+    )
+    regression_problem = problems.RegressionProblem(
+        observations=obs, locations=times, solution=states
+    )
+    return dynmod, measmod, initrv, regression_problem
+
+
+def ornstein_uhlenbeck(delta_t=0.2, lam=0.21, dynvar=0.5, measvar=0.1, t_max=20.0):
+    r"""State space model based on an Ornstein Uhlenbeck process.
+
+    A linear, time-invariant state space model for the dynamics of a time-invariant
+    Ornstein-Uhlenbeck process. See e.g. Example 10.19 in Särkkä et. al, 2019. [1]_
+    Here, we formulate a continuous-discrete state space model:
+
+    .. math::
+
+        x(t_n) &= \lambda x(t_{n-1}) + q(t_n) \\
+        y_n &= x(t_n) + r_n
+
+    for some scalar :math:`\lambda` and :math:`q(t_n) \sim \mathcal{N}(0, Q)`,
+    :math:`r_n \sim \mathcal{N}(0, R)` for process noise covariance matrix :math:`Q`
+    and measurement noise covariance matrix :math:`R`.
+    Note that the linear, time-invariant dynamics have an equivalent discretization.
+
+    Parameters
+    ----------
+    delta_t : float
```

Comment:
instead of delta_t, I'd advocate for either `step` (which is what is currently used in the old parts of probnum) or `dt` (which is what I'd like to have everywhere soon, and is used inside statespace for instance).

Reply:
See issue #346